doc/CMakeLists.txt

```diff
@@ -74,11 +74,6 @@ add_custom_target(
   COMMENT "Running ${DOXYGEN_EXECUTABLE}"
 )
 
-add_custom_target(
-  pristine
```

Comment:
Pristine does a bit more than just a `ninja / make clean` as it does a complete pristine, that is removing also files like CMakeCache.txt so that the complete build starts over.

Could you elaborate a bit more (in commit message perhaps or a dedicated issue), or what is the problem with the pristine target.

Currently it just states:
>  doc: cmake: remove pristine target
>
> Proper clean targets will be added.

but not why having `pristine` is wrong.

Reply:
I'll add it back then